src/js/components/Header/VolumeControl.react.js

```diff
@@ -65,7 +65,7 @@ export default class VolumeControl extends Component {
 
     setVolume(e) {
         AppActions.player.setVolume(parseFloat(e.currentTarget.value));
-        this.setState({ volume: e.currentTarget.value });
+        this.setState({ volume: Math.pow(e.currentTarget.value, 4) });
```

Comment:
Hmm. I think this commit re-breaks exponential volume control. This line will set the state to the correct value, but line 67 is the one that actually changes the audio volume, right?